.env

```diff
@@ -60,3 +60,8 @@ DATABASE_URL="mysql://root@127.0.0.1:3306/packagist?serverVersion=8&charset=utf8
 APP_ENV=dev
 APP_SECRET=f7bd3db39aea6b0c35d6703fab361df5
 ###< symfony/framework-bundle ###
+
+###> web-auth/webauthn-symfony-bundle ###
+RELAYING_PARTY_ID=packagist.org
```

Comment:
This should be `Relying`, not `Relaying` (if this comes from a recipe, the naming might need to be fixed in it)

Reply:
🫢https://github.com/symfony/recipes-contrib/pull/1513